packages/@glimmer/runtime/test/initial-render-test.ts

```diff
@@ -46,14 +46,33 @@ class AbstractRehydrationTests extends InitialRenderSuite {
     this.assert.equal(clearedNodes.length, nodes, 'cleared nodes');
   }
 
-  assertServerOutput(..._expected: Content[]) {
+  assertExactServerOutput(..._expected: Content[]) {
     let output = expect(this.serverOutput, 'must renderServerSide before calling assertServerOutput');
-    equalTokens(output, content([OPEN, ..._expected, CLOSE]));
+    equalTokens(output, content([..._expected]));
```

Comment:
I think it would be better if we avoided a second `content` wrapping here.  That means we expect "exact contents" to be passed in (but thats what the method name infers so it seems good?)...

Reply:
Done!  Good catch!